ClientTests/SearchTests.swift

```diff
@@ -45,12 +45,6 @@ class SearchTests: XCTestCase {
         checkInvalidURL("创业咖啡. 中国")
     }
 
-    func testURIFixupPunyCode() {
-        checkValidURL("http://创业咖啡.中国/", afterFixup: "http://xn--vhq70hq9bhxa.xn--fiqs8s/")
```

Comment:
Just leaving comment for clarification, this is handled by webkit now, and no longer needed in our code base, so URL fixing up, is not handled by us.